pkg/apis/nmstate/v1alpha1/zz_generated.openapi.go

```diff
@@ -11,13 +11,13 @@ import (
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"./pkg/apis/nmstate/v1alpha1.Condition":                            schema_pkg_apis_nmstate_v1alpha1_Condition(ref),
-		"./pkg/apis/nmstate/v1alpha1.NodeNetworkConfigurationPolicy":       schema_pkg_apis_nmstate_v1alpha1_NodeNetworkConfigurationPolicy(ref),
-		"./pkg/apis/nmstate/v1alpha1.NodeNetworkConfigurationPolicySpec":   schema_pkg_apis_nmstate_v1alpha1_NodeNetworkConfigurationPolicySpec(ref),
-		"./pkg/apis/nmstate/v1alpha1.NodeNetworkConfigurationPolicyStatus": schema_pkg_apis_nmstate_v1alpha1_NodeNetworkConfigurationPolicyStatus(ref),
-		"./pkg/apis/nmstate/v1alpha1.NodeNetworkState":                     schema_pkg_apis_nmstate_v1alpha1_NodeNetworkState(ref),
-		"./pkg/apis/nmstate/v1alpha1.NodeNetworkStateStatus":               schema_pkg_apis_nmstate_v1alpha1_NodeNetworkStateStatus(ref),
-		"./pkg/apis/nmstate/v1alpha1.State":                                schema_pkg_apis_nmstate_v1alpha1_State(ref),
+		"github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1.Condition":                            schema_pkg_apis_nmstate_v1alpha1_Condition(ref),
```

Comment:
That's unexpected

Reply:
its fine but ? guess just something that wasnt auto generated before
path at my laptop is ~/go/src/github.com/nmstate/kubernetes-nmstate

Reply:
This is indeed weird, granted Quique did the same operation recently and it kept `./`.

Reply:
Quique, do you use same path for the git clone ?
(~/go/src/github.com/nmstate/kubernetes-nmstate)

Reply:
So on my environment it keeps what Quique has if I don't export GOPATH. If I do export it, it changes it to the same as it does to Or.

@qinqon did you have proper exports before running the generate script?

Reply:
let's not depend on GOPATH since is going to be deprecated.
Or ensure the following that you don't have GOPATH before running
we can even clean export at Makefile.